sirenabailie/Fan-Fusion-FE
# Story details2

## Detailed Description
Added a Story Details page that shows story name, author username, and story description.

## Related Issues
Table of Contents and Tags

## How to Test
Pull down StoryDetails2 branch, and run on local server. Click view button on stories to see story details

## Motivation and Context


## Screenshots (if applicable)

## Types of Changes

- [x ] ✨ New feature

## 💬 Additional Notes
